src/Roles/Role.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace ATDev\RocketChat\Roles;
+
+use ATDev\RocketChat\Common\Request;
+use ATDev\RocketChat\Users\User;
+
+/**
+ * Role class
+ */
+class Role extends Request
+{
+    use \ATDev\RocketChat\Roles\Data;
+
+    /**
+     * Gets role listing
+     *
+     * @return Collection|bool
+     */
+    public static function listing()
+    {
+        static::send("roles.list", "GET");
+
+        if (!static::getSuccess()) {
+            return false;
+        }
+
+        $roles = new Collection();
+
+        foreach (static::getResponse()->roles as $role) {
+            $roles->add(static::createOutOfResponse($role));
+        }
+
+        return $roles;
+    }
+
+    /**
+     * Gets all the roles in the system which are updated after a given date
+     *
+     * @return $this|false
+     */
+    public function sync()
+    {
+        static::send("roles.sync", "GET", ["updatedSince" => $this->getUpdatedSince()]);
+        if (!static::getSuccess()) {
+            return false;
+        }
+
+        $roles = static::getResponse()->roles;
+
+        $this->update = new Collection();
```

Comment:
@cvalex It's not a property of the role and should be dropped. The method should return just an array of `update` and `remove` keys with collection of roles.